config/routes.rb

```diff
@@ -3,4 +3,11 @@
 Rails.application.routes.draw do
   get "health_check" => "application#health_check"
   root to: "visitors#index"
+
+  get "dashboard" => "dashboard#show"
+
+  get "auth/oauth2/callback" => "auth0#callback"
+  get "auth/failure" => "auth0#failure"
+
+  get "logout" => "logout"
```

Comment:
To go with the earlier comment about using a different HTTP method for signing out 
```suggestion
  delete "logout" => "logout"
```